CHANGELOG.md

```diff
@@ -17,6 +17,12 @@ Dates are in `yyyy-mm-dd`.
 * Incorrect theme of system navigation bar
 * No refresh indicator when opening notifications
 
+### Changed
+* ProgressDialog in TokenActivity to on screen prompts
+
+### Changed
+* ProgressDialog in TokenActivity with on screen prompts
+
```

Comment:
These should come under Unreleased.  Changes under a version number are released changes.
Read [this](https://github.com/crux-bphc/CMS-Android/blob/development/CONTRIBUTING.md) and [this](https://keepachangelog.com/en/1.0.0/) to learn more.

Reply:
Also, the same changelog (+ heading) has duplicated.
Also also, changelog should be written for the end user. They don't know what a `ProgressDialog` is. A more appropriate changelog is _A new and modern looking and more performance Login Screen_ or something, because at the end of the day, it is just a cosmetic change for the end user (irrespective of how relavant the technical changes might have been)